src/constants/networked-trajectories.ts

```diff
@@ -123,6 +135,136 @@ const TRAJECTORIES: TrajectoryDisplayData[] = [
         imageFile: smoldynSpineImage,
         gifFile: smoldynSpineGif,
     },
+    {
+        modelName: "Actin Bundle Dynamics with α–Actinin and Myosin",
+        id: "medyan_Chandrasekaran_2019_UNI_alphaA_0.1_MA_0.0225.simularium",
+        title: "Actin Bundle Dynamics with α–Actinin and Myosin",
+        subtitle: "Low Myosin Activity",
+        totalSimulatedTime: "33min",
+        authors: "Chandrasekaran et al.",
+        publication: {
+            title:
+                "Remarkable structural transformations of actin bundles are driven by their initial polarity, motor activity, crosslinking, and filament treadmilling",
+            journal: "PLoS Computational Biology",
+            year: 2019,
+            url:
+                "https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007156",
+        },
+        description:
+            "A MEDYAN model of interacting actin filaments, α–actinin crosslinkers, and myosin motors. Low myosin activity maintains the bundle structure.",
+        code:
+            "Software used to generate this data is available <a href='https://bitbucket.org/jkomianos/medyan/src/master/'>here</a>. The outputs that were visualized can be downloaded <a href='https://drum.lib.umd.edu/handle/1903/21856'>here</a>.",
+        legalese:
+            "Please note that the use of this model is subject to third party licensing requirements, which are currently available <a href='https://bitbucket.org/jkomianos/medyan/src/master/license.txt'>here</a>.",
+        imageFile: medyan1Image,
+        gifFile: medyan1Gif,
+    },
+    {
+        modelName: "Actin Bundle Dynamics with α–Actinin and Myosin",
+        id: "medyan_Chandrasekaran_2019_UNI_alphaA_0.1_MA_0.675.simularium",
+        title: "Actin Bundle Dynamics with α–Actinin and Myosin",
+        subtitle: "High Myosin Activity",
+        totalSimulatedTime: "33min",
+        authors: "Chandrasekaran et al.",
+        publication: {
+            title:
+                "Remarkable structural transformations of actin bundles are driven by their initial polarity, motor activity, crosslinking, and filament treadmilling",
+            journal: "PLoS Computational Biology",
+            year: 2019,
+            url:
+                "https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007156",
+        },
+        description:
+            "A MEDYAN model of interacting actin filaments, α–actinin crosslinkers, and myosin motors. High myosin activity causes formation of an aster-like structure.",
+        code:
+            "Software used to generate this data is available <a href='https://bitbucket.org/jkomianos/medyan/src/master/'>here</a>. The outputs that were visualized can be downloaded <a href='https://drum.lib.umd.edu/handle/1903/21856'>here</a>.",
+        legalese:
```

Comment:
this seems like something that can be factored out since it's repeated

Reply:
Okay, I made a function returning a template literal for the legalese. I didn't do anything with the `code` text because that one varies slightly, and I didn't want to make it too complicated especially if we're going to make this all part of the trajectory file metadata soon ;)